### DISCUSSION TITLE
Bedrock or Java

### DISCUSSION THREAD

**Discussion by jimmy927** [PrismarineJS/mineflayer#3173] [Q&A] [answered]
I am trying to get started.
But I am getting very confused about very very basic information missing.

Is this for bedrock or java or both ?

**Comment by rom1504** [comment, marked as answer]
Currently java = pc

**Reply by jimmy927** [reply]
I am note sure I understand.

If it is for java why would it not work on Mac & Linux ?

**Reply by rom1504** [reply]
java version is called pc
